modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/util/CalendarUtil.java

```diff
@@ -319,7 +320,7 @@ public static JSONObject toCalendarJSONObject(
 			"calendarResourceId", calendarResource.getCalendarResourceId()
 		).put(
 			"calendarResourceName",
-			HtmlUtil.escape(calendarResource.getName(themeDisplay.getLocale()))
+			calendarResource.getName(themeDisplay.getLocale())
```

Comment:
Why did you extract the `escape` here?

Reply:
After talking in person, this was reverted.